backend/api/Database/Context/FlotillaDbContext.cs

```diff
@@ -111,27 +100,17 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             modelBuilder.Entity<Plant>().HasIndex(a => new { a.PlantCode }).IsUnique();
 
             modelBuilder
-                .Entity<Area>()
-                .HasOne(a => a.InspectionArea)
-                .WithMany()
-                .OnDelete(DeleteBehavior.Restrict);
-            modelBuilder
-                .Entity<Area>()
-                .HasOne(a => a.Plant)
-                .WithMany()
-                .OnDelete(DeleteBehavior.Restrict);
-            modelBuilder
-                .Entity<Area>()
-                .HasOne(a => a.Installation)
-                .WithMany()
-                .OnDelete(DeleteBehavior.Restrict);
+                .Entity<InspectionGroup>()
+                .HasIndex(a => new { a.Name, a.Installation })
+                .IsUnique();
+
             modelBuilder
-                .Entity<InspectionArea>()
-                .HasOne(d => d.Plant)
+                .Entity<InspectionGroup>()
+                .HasOne(d => d.Installation)
```

Comment:
d (for deck) should probably be changed as well. Have notices a few of those in frontend as well